<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>855</width>
    <height>649</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Soft Potato 2.0</string>
  </property>
  <property name="windowIcon">
   <iconset resource="icon.qrc">
    <normaloff>:/mainLogo/logo.svg</normaloff>:/mainLogo/logo.svg</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="1">
     <widget class="PlotWidget" name="plot2" native="true"/>
    </item>
    <item row="4" column="0" colspan="2">
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="1" column="0">
       <widget class="QProgressBar" name="progressBar">
        <property name="value">
         <number>0</number>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="2">
       <widget class="QSlider" name="slider_plots">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="value">
         <number>99</number>
        </property>
        <property name="sliderPosition">
         <number>99</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="2" column="0">
     <widget class="PlotWidget" name="plot3" native="true"/>
    </item>
    <item row="1" column="0">
     <widget class="PlotWidget" name="plot1" native="true"/>
    </item>
    <item row="2" column="1">
     <widget class="PlotWidget" name="plot4" native="true"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>855</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <widget class="QMenu" name="menuSave">
     <property name="title">
      <string>Save</string>
     </property>
     <addaction name="action_tEi"/>
     <addaction name="action_O"/>
     <addaction name="action_R"/>
     <addaction name="action_x"/>
     <addaction name="separator"/>
     <addaction name="action_Save_all"/>
    </widget>
    <addaction name="menuSave"/>
    <addaction name="separator"/>
    <addaction name="fileExit"/>
   </widget>
   <widget class="QMenu" name="menuTechnique">
    <property name="title">
     <string>Technique</string>
    </property>
    <addaction name="actionCyclic_voltammetry"/>
    <addaction name="actionChronoamperometry"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionHelp"/>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuSimulation">
    <property name="title">
     <string>Simulation</string>
    </property>
    <addaction name="actionSimulate"/>
    <addaction name="separator"/>
    <addaction name="actionArea"/>
    <addaction name="actionMechanism"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuTechnique"/>
   <addaction name="menuSimulation"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionCyclic_voltammetry">
   <property name="text">
    <string>Cyclic voltammetry</string>
   </property>
   <property name="shortcut">
    <string>Alt+V</string>
   </property>
  </action>
  <action name="actionLinear_sweep_voltammetry">
   <property name="text">
    <string>Linear sweep voltammetry</string>
   </property>
  </action>
  <action name="actionE">
   <property name="text">
    <string>E</string>
   </property>
  </action>
  <action name="action_tEi">
   <property name="text">
    <string>t, E, i</string>
   </property>
  </action>
  <action name="action_O">
   <property name="text">
    <string>[O]</string>
   </property>
  </action>
  <action name="action_R">
   <property name="text">
    <string>[R]</string>
   </property>
  </action>
  <action name="fileExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionChronoamperometry">
   <property name="text">
    <string>Chronoamperometry</string>
   </property>
   <property name="shortcut">
    <string>Alt+A</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionGrid">
   <property name="text">
    <string>Grid</string>
   </property>
  </action>
  <action name="action_x">
   <property name="text">
    <string>x</string>
   </property>
  </action>
  <action name="actionSimulate">
   <property name="text">
    <string>Simulate</string>
   </property>
   <property name="shortcut">
    <string>F5</string>
   </property>
  </action>
  <action name="actionMechanism">
   <property name="text">
    <string>Mechanism</string>
   </property>
   <property name="shortcut">
    <string>Alt+M</string>
   </property>
  </action>
  <action name="actionArea">
   <property name="text">
    <string>Area</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="text">
    <string>Help</string>
   </property>
  </action>
  <action name="action_Save_all">
   <property name="text">
    <string>Save all...</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlotWidget</class>
   <extends>QWidget</extends>
   <header>pyqtgraph</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="icon.qrc"/>
 </resources>
 <connections/>
</ui>
